addons/godot-xr-tools/functions/movement_climb.gd

```diff
@@ -156,17 +155,17 @@ func _average_velocity() -> Vector3:
 	# Return the average
 	return total_distance / total_time
 
-# This method verifies the MovementProvider has a valid configuration.
+# This method verifies the movement provider has a valid configuration.
 func _get_configuration_warning():
 	# Verify the left controller
 	var test_left_pickup_node = get_node_or_null(left_pickup) if left_pickup else null
-	if !test_left_pickup_node or !test_left_pickup_node is Function_Pickup:
-		return "Unable to find left Function_Pickup"
+	if !test_left_pickup_node or !test_left_pickup_node is XRToolsFunctionPickup:
+		return "Unable to find left FunctionPickup"
```

Comment:
Shouldn't the error also have the full name `XRToolsFunctionPickup` ?

Reply:
Modified to use correct class name.